app/models/slide.rb

```diff
@@ -70,7 +70,7 @@ def page_list_url
   def page_list
     len = num_of_pages.abs.to_s.length
     result = []
-    for i in 1..num_of_pages
+    (1..num_of_pages).each do |i|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'i' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)